content/axon-server-installation-guides/modules/ROOT/pages/installing_as_docker-k8s.adoc

```diff
@@ -0,0 +1,604 @@
+= Installing Axon Server for Docker and Kubernetes
+
+pass:[<!-- vale Google.Passive = NO -->]
+pass:[<!-- vale Google.Will = NO -->]
+pass:[<!-- vale Google.We = NO -->]
+pass:[<!-- vale Google.FirstPerson = NO -->]
+pass:[<!-- vale AxonIQ.AcronymCase = NO -->]
+pass:[<!-- vale AxonIQ.Headings = NO -->]
+
+This section is split into 4 sub-sections.
+
+* <<docker-image,The Axon Server Docker Image>>
+* <<build-image,Construction of the Axon Server Docker Image>>
+* <<docker-compose,Deployment using Docker Compose>>
+* <<kubernetes,Deployment using Kubernetes>>
+
+____
+
+Unified Axon Server Artifact 
+
+In an effort to simplify the deployment process, since 2023.1 version we've updated Axon Server to release a single artifact, replacing the previously separate artifacts for the Standard and Enterprise edition.
+Depending on the presence of a license, Axon Server will now automatically adjust.
+
+____
+
+== Docker image [[docker-image]]
+
+AxonIQ provides ready to use https://hub.docker.com/r/axoniq/axonserver[Axon Server images].
+There are two types of images available: one with Axon Server running as the user "`root`" and one with Axon Server running as user "`axonserver`".
+Both images are based on https://hub.docker.com/_/eclipse-temurin[Eclipse Temurin], which means they include a (limited) shell that allows you to connect "into" the running image and perform some commands.
+
+* The "`root`" image of version 2023.1 is available as "`axoniq/axonserver:2023.1`" and is based on "`eclipse-temurin:11-focal`".
```

Comment:
In fact, if we move it to the axon-server, we can directly use a ${project.version}  property in the docs so that for every build, the docs are aligned with the property from the source code.